scene/2d/screen_button.cpp

```diff
@@ -135,7 +135,7 @@ void TouchScreenButton::_notification(int p_what) {
 			update();
 
 			if (!get_tree()->is_editor_hint())
-				set_process_input(true);
+				set_process_input(is_visible());
```

Comment:
I think this should be `is_visible_in_tree()`.
Otherwise it will still keep processing when the button itself is "visible" but implicitly hidden via a hidden parent node.